lib/index.ts

```diff
@@ -146,6 +149,35 @@ async function dispatchCommand(
         }
     }
 }
+const handleFloorPriceChannel = async (client: any, magicEdenData: any)  => {
+    try {
+        const myChannel = client.channels.cache.get(config.priceChannel);
+        const price = Number(magicEdenData.floorPrice)/LAMPORTS_PER_SOL
+        myChannel.setName(`Floor Price: ◎${price}`);
+    } catch(error) {
+        console.log(error);
+    }
+
+};
+
+const handleTotalVolumeChannel = async (client: any, magicEdenData: any)  => {
+    try {
+        const myChannel = client.channels.cache.get(config.volumeChannel);
+        const volume = numberWithCommas((Math.round(magicEdenData.volumeAll/LAMPORTS_PER_SOL)).toString())
```

Comment:
Number with commas already returns a string I don't think the .toString() is needed